libcontainer/rootfs_linux.go

```diff
@@ -572,6 +572,13 @@ func pivotRoot(rootfs, pivotBaseDir string) error {
 	if err != nil {
 		return fmt.Errorf("can't create pivot_root dir %s, error %v", pivotDir, err)
 	}
+	defer func() error {
+		err := os.Remove(pivotDir)
+		if err != nil {
+			return err
+		}
+		return nil
+	}()
```

Comment:
@hqhq 
Even if the pivotBse is empty, function is initializing with / 
if pivotBaseDir == "" {
		pivotBaseDir = "/"
	}
Post that the directory is created, is there any case where this file/directory will not be created ?